wordpress-mobile/WordPress-Android
# Issue/start over button

### Fix
Update the start over preference layout to avoid clipping the bottom button shadow.  See the screenshots below for illustration.

![start_over](https://user-images.githubusercontent.com/3827611/52548464-0dfe8980-2d8b-11e9-86c4-e2d90c379228.png)

### Test
0. Select site with ***Personal***, ***Premium***, or ***Business*** plan.
1. Go to ***Sites*** tabs.
2. Tap ***Settings*** under ***Configuration*** section.
3. Tap ***Start Over*** under ***Advanced*** section.
4. Long-press ***Contact Support*** button.
5. Notice button shadow is not clipped.

### Review
Only one developer is required to review these changes, but anyone can perform the review.
